plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/utils/ConfigurationUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.utils;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public final class ConfigurationUtils
+{
+    private ConfigurationUtils() {}
+
+    public static ImmutableMap<String, String> readProperties(List<File> resourcePaths)
```

Comment:
Please write unit tests for this method? If several properties for the same key are found do we fail or overwrite?

Use `Map` as return type.

Reply:
Added a unit test which shows that the properties with the same key will be overwritten by the last provided file.